Action.c

```diff
@@ -438,7 +447,7 @@ static const struct {
    { .key = "      H: ", .info = "hide/show user process threads" },
    { .key = "      K: ", .info = "hide/show kernel threads" },
    { .key = "      F: ", .info = "cursor follows process" },
-   { .key = "    + -: ", .info = "expand/collapse tree" },
+   { .key = "  + - *: ", .info = "expand/collapse tree" },
```

Comment:
Include some indication, that <kbd>*</kbd> does all branches, in contrast to the other two keys.